### PR TITLE
Adds an SCP facility lavaland ruin

### PR DESCRIPTION
**Pull request by matskuman5**
Adds a new lavaland ruin. It's an anomalous object containment site run by NanoTrasen. It was hit by an earthquake a few decades ago and it has fallen to ruin. It holds some dangerous creatures inside, but it's got an open armoury at the end as a reward.

<a href="http://imgur.com/uBfTd54"></a>

Includes fluff object testing logs!

:cl:
rscadd: A new lavaland ruin has been unearthed!
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
> SCP
> No SCP's on the map
> 0 New mobs

**Comment by matskuman5** [comment]
I dont have the coding / spriting skills bby :(

**Comment by Slinky5890** [comment]
Once some mobs are sorted out (maybe even just a living statue that activates as a ghost role if someone enters, dunno how doable that is) then i'd definitely be interested in this.

File: yogstation.dme [hunk 1]
```diff
@@ -15,6 +15,8 @@
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\tgstation2.dm"
+#include "_maps\RandomRuins\LavaRuins\lavaland_surface_scp_facility.dmm"
+#include "_maps\RandomRuins\SpaceRuins\emptyshell.dmm"
```
Comment (monster860) [inline yogstation.dme]: **AHHH NO THAT IS NOT HOW YOU ADD RUINS!!!!**

**Review by monster860 — CHANGES_REQUESTED** [review]

**Comment by Stealthkibbler** [comment]
Needs skeletal remains and human remains.

If you cant make any new mobs then dig around in simple_animals for anything that could even slightly add to the ruins. 

As long as its fun for miners to explore and has something cool to find then it generally will be fine, yogs doesn't have any defining expectations or restrictions.

**Comment by monster860** [comment]
@matskuman5 

https://github.com/yogstation13/yogstation/blob/master/code/datums/ruins/lavaland.dm

Add your ruin here, revert your yogstation.dme back to what it was before.

**Comment by Stealthkibbler** [comment]
The paper notes are cool though 👍

**Comment by ghost** [comment]
>kill one old asteroid goliath and not lavaland goliath

>kill one legion

>GET FUCKING AUTORIFLES

are you sERIOUS

**Comment by matskuman5** [comment]
You missed the buffed eye carp and the faithless, Adam

**Comment by Alek2ander** [comment]
You realise miners can... mine? And don't really have a problem with rwalls. It's piss easy to get the loot while avoiding all hazards

**Comment by RoboticVodka** [comment]
Seriously you only have to just walk around.... Even if you can't get past the Rwalls just go get an RCD, and make a bridge across the lava.

**Comment by matskuman5** [comment]
Made the walls unbreakable so you have to go through the enemies to get to the armoury. Added an imp and a christmas tree, plus some skeletal and normal corpses

**Comment by Time-Green** [comment]
Could you ask someone to resprite a statue into The Sculpture?
[COPYRIGHT]
I mean, if you're making an SCP facility...

**Comment by Dragonforcefive** [comment]
Lol. I can just imagine miners getting better weapons then security.

**Comment by Cruix** [comment]
How did you manage to post the only image on that entire massive website that is not public domain?

**Review by monster860 — CHANGES_REQUESTED** [review]
https://github.com/yogstation13/yogstation/blob/master/code/datums/ruins/lavaland.dm

Add your ruin here, revert your yogstation.dme back to what it was before.

**Comment by Stealthkibbler** [comment]
As monster pointed out, ruins are not defined in the dme but rather in lavaland.dm

**Comment by ghost** [comment]
AUTORIFLES

**Comment by Super3222** [comment]
Looks nice, but the loot might be a problem. What type of guns are those in the armory?

**Comment by matskuman5** [comment]
Yeah this is fucked, I'll probably redo this later but can't be arsed to fix it now

I'll close this

**Pull request closed without merging**